### ISSUE TITLE
Suggest accessibility: items sometimes announced with a general name

### ISSUE THREAD

**Issue by isidorn** [Microsoft/vscode#3862]
Names of suggestions in the suggest widget are sometimes not announced, instead I get 'list', 'list item' or 'unknown'. Here's an example on how to repro:
1. vscode repo, open `debugService.ts`:119
2. `this.` trigger suggest widget - go through all the items and notice that not all of them will be read out, some will have the 'unknown', 'list'
3. coming back to the item, the correct name sometimes gets read out

I guess this is probably a bug in chrome or nvda

**Comment by alexandrudima** [comment]
I have removed the proper way (using aria-autocomplete and aria-activedescendant) and implement this now using aria alerts due to https://bugs.chromium.org/p/chromium/issues/detail?id=593646
